TestingGrounds/TestingGrounds/HashTable.cpp

```diff
@@ -1,2 +1,92 @@
 #include "HashTable.hpp"
 
+template <typename K, typename V>
+HashTable<K, V>::HashTable()
+{
+    capacity { 20 };
```

Comment:
How did this compile?

Reply:
I don't think you can utilize uniform initialization here since these are declared else where.

Reply:
UId